src/client/results/MostUsedWords.js

```diff
@@ -89,6 +79,28 @@ const WordItem = ({ wordInfo }) => (
   </div>
 );
 
+let WordItem = ({ dispatch, makeupInfo, conflatedWordCount }) => (
+  <tr>
+    <td className='right aligned' style={{ width: '60px' }}>{Math.round(makeupInfo.count / conflatedWordCount * 100)}%</td>
+    <td className='right aligned' style={{ width: '60px' }}>{makeupInfo.count}</td>
+    <td>{makeupInfo.word}</td>
+    <td>
+      <div className="ui checkbox" onClick={(e) => {
+        const isNowChecked = $($(e.target).parent().children()[0]).is(':checked');

```

Comment:
This makes me cry.

I'm getting the checked value of the checkbox - but it appears no matter where you click, it's always the `<label>Show</label>` that seems to pick up the event? Hence getting its parent, then its children, then the input `[0]` - even though it should be `[1]`? o.O